pkg/sdk/testint/views_gen_integration_test.go

```diff
@@ -3,15 +3,16 @@ package testint
 import (
 	"errors"
 	"fmt"
+	"sort"
 	"testing"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-// TODO [SNOW-1016430]: add tests for setting masking policy on creation
 // TODO [SNOW-1016430]: add tests for setting recursive on creation
 func TestInt_Views(t *testing.T) {
 	client := testClient(t)
```

Comment:
why no new assertions?

Reply:
Ah, I'll use them in resource implementation.

Reply:
Okay, and also use them here (even in a separate PR).

Reply:
Used here.